docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -1521,3 +1521,9 @@ with Parquet files performed by the Iceberg connector.
         for structural data types. The equivalent catalog session property is
         ``parquet_optimized_nested_reader_enabled``.
       - ``true``
+    * - ``parquet.use-bloom-filter``
```

Comment:
This can be squashed into previous commit

Reply:
done.